packages/core/src/decorators/common/media-type-examples-override.ts

```diff
@@ -0,0 +1,41 @@
+import { Oas3Decorator } from '../../visitors';
+import { Oas3Operation, Oas3RequestBody } from '../../typings/openapi';
+import { readAndParseFileSync } from '../../utils';
+
+export const MediaTypeExamplesOverride: Oas3Decorator = ({ operationIds }) => {
+  let operationId: string | undefined;
+  return {
+    Operation: {
+      leave(operation: Oas3Operation) {
+        operationId = operation.operationId;
+        const properties = operationIds[operationId!];
+
+        if (properties.responses && operation.responses) {
+          for (const responseCode of Object.keys(properties.responses)) {
+            if (!operation.responses[responseCode] || !operation.responses[responseCode]?.content) {
+              continue;
+            }
+
+            Object.values(operation.responses[responseCode].content!).forEach((mimeType) => {
```

Comment:
Could you use either `forEach` or `for ... of` here and [here](https://github.com/Redocly/redocly-cli/pull/897/files#diff-150f23c15f2f75bc7e9556b690b68cd5c77fa61ce8801d6fd346ea86775c26feR33)?